docs/user/reference/sitemaps.rst

```diff
@@ -0,0 +1,38 @@
+``sitemap.xml``  support
+========================
+
+`Sitemaps <https://www.sitemaps.org/>`__ allow you to inform search engines about URLs that are available for crawling.
+This helps make your content more discoverable,
```

Comment:
```suggestion
This makes your content more :term:`discoverable <discoverability>`,
```

Reply:
Nice -- I had that originally, but it errored because it wasn't merged yet :)